infra/scripts/cleanup_dynamo_ci.py

```diff
@@ -0,0 +1,16 @@
+import boto3
+
+
+def main() -> None:
+    db = boto3.resource("dynamodb")
+
+    num_deleted = 0
+    for table in db.tables.all():
+        if "integration_test" in table.name:
+            table.delete()
+            num_deleted += 1
+    print(f"Deleted {num_deleted} CI DynamoDB tables")
```

Comment:
Probably nicer to use tqdm here if not too much effort

Reply:
done